src/index.js

```diff
@@ -1,4 +1,3 @@
-import _ from 'lodash';
 import './style.css';
 
 const inputText = document.querySelector('input');
```

Comment:
- [ ] There should be an array of different objects with keys `description(string)`, `completed(boolean)`, `index(integer)`.

  The objects in this array should be dynamically rendered on the HTML page using the `add` function you have in this file.

Reply:
@TedLivist I created and array I think you didn't see that

Reply:
@fahimy143 I can see the array but it is empty. I am on this branch. See

![fahimy-1](https://user-images.githubusercontent.com/36542986/177562149-abba3997-ab28-41f5-9ad0-df9431e057ed.jpg)

Reply:
![image](https://user-images.githubusercontent.com/57341400/177562578-9bff5f92-a156-427a-85fd-e1ffe02ff226.png)
I created an object to push inside that empty array

Reply:
@fahimy143 at this point in the project you are required to have **hardcoded** objects in this array. The values of these objects should be dynamically rendered on the HTML page.

Reply:
@fahimy143 yes, not from the input yet. Task addition or removal will be implemented in the next project, I think.

Reply:
@fahimy143 I can only request changes for this specific project. Or did you intend to request changes for the add-remove project on your dashboard?

You can request a second opinion if you are not satisfied with the requested changes. Here is the [link to request the S.O.](https://airtable.com/shrQAqnBwek5a0O0s)